components/dashboard/src/components/DropDown2.tsx

```diff
@@ -229,11 +237,11 @@ export const DropDown2: FunctionComponent<DropDown2Props> = ({
                                     </li>
                                 );
                             })
-                        ) : (
+                        ) : !loading ? (
                             <li key="no-elements" className={"rounded-md "}>
                                 <div className="h-12 pl-8 py-3 text-gray-800 dark:text-gray-200">No results</div>
```

Comment:
**issue(non-blocking)**: Would be nice to have a nudge here to check with provider permissions, connect more providers, etc. However, let's leave this out of the scope of this PR.

<img width="507" alt="Screenshot 2023-09-14 at 20 11 59" src="https://github.com/gitpod-io/gitpod/assets/120486/d92a3d38-859b-415d-9881-4a156079ecff">